src/subscriber.rs

```diff
@@ -216,7 +216,21 @@ where
             .fetch_sub(1, atomic::Ordering::AcqRel)
         {
             // span has ended!
-            // reclaim its id
+            if self.timing.span_close_events {
+                // record a span-end event
```

Comment:
Won't this end up double-counting with `on_close`?

Reply:
Assuming you mean [`on_close`](https://docs.rs/tracing-subscriber/0.2.17/tracing_subscriber/layer/trait.Layer.html#method.on_close), from my understanding the subscriber and layer won't be active at the same time? So, if the subscriber is in play then `try_close` is the only way to know about closes, and if the layer is active then `on_close` is.

Reply:
@hawkw can you confirm that that is indeed the case?